.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Gem release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
We should try to use SHA values instead of the version for github actions, due to security hardening and to avoid any issues if these actions have any releases/updates.

Github Action best practices: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
You are right. I just went with what we usually have. I can change this.